### DISCUSSION TITLE
추가적인 기능 구현 - 알림

### DISCUSSION THREAD

**Discussion by Hyevvy** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#215] [General]
# 알림목록 
- 내가 작성한 게시글에 댓글이 달렸을 때 
- `내가 작성한 글에 댓글이 달렸습니다` -> 누르면 detail/postId로 이동
- 벨 누르는 순간 다 읽었다고 api 쏘기 
- 드롭다운으로 하고 스크롤할 지 / 몇 개 자를지

#

**Comment by YuHyun-P** [comment]
> 드롭다운으로 하고 스크롤할 지 / 몇 개 자를지

최신 알림 3개만 보여주는 걸로 결정! 땅땅땅!
